src/components/NavBar/NavBar.jsx

```diff
@@ -90,16 +108,76 @@ const NavBar = () => {
             {session.info.isLoggedIn ? (
               <>
                 <Box>
+                  {/* ------------START----------- */}
```

Comment:
Potential refactor/pull this out into its own component.

Reply:
Complete -> NavbarLinks.jsx